layouts/partials/head.html

```diff
@@ -55,6 +60,8 @@
   {{ $is_blog := and (eq .Type "blog") (eq .Kind "page") }}
   {{ $has_image :=  isset .Params "banner" }}
   {{ $image := cond $has_image .Params.banner .Site.Params.default_sharing_image }}
+  {{ $is_valid_image := fileExists $image }}
```

Comment:
without .Params.banner .Site.Params.default_sharing_image I have the following error:

```bash
executing "partials/head.html" at <fileExists $image>: error calling fileExists: fileExists needs a path to a file
```

Reply:
fixed by https://github.com/devcows/hugo-universal-theme/pull/239/commits/1fd2946dbdca274ccd20ae11f06bdf245fb0cd53